DEVELOPERS.md

```diff
@@ -98,3 +98,66 @@ Each category lives in its own directory (for example `tests/unit`) and can be r
 Additional arguments passed to `just test` are passed on to pytest. For example, to
 run all the tests _except_ the functional tests, run `just test -k 'not functional'`,
 or to run a single test, run e.g. `just test tests/unit/test_urls.py::test_urls`.
+
+
+# Local job-server for integration.
+
+## First time set up
+
+This needs some first time setup, but after that is fairly simple to use. You
+will need the Bitwarden cli `bw` installed to pull the dev Github auth
+credentials. You need to run the following command with your github username.
+
+```
+just job-server/configure GHUSERNAME
+```
+
+This will configure and run the latest job-server image at
+http://localhost:9000 to use in integration testing airlock. It will
+automatically point your current .env config to this local instance.
+
+This command is idempotent, and can be safely re-run.
+
+
+In future you can just do the following to start it up:
+
+```
+just job-server/run
+```
+
+## Create workspace
+
+You will need at least one workspace set up in job-server and locally in airlock to test integration:
+
+
+```
+just job-server/create-workspace NAME  # defaults to "airlock-test-workspace"
```

Comment:
I ran this with the default, so it created me an `airlock-test-workspace` on job-server, but then I don't have one on airlock because the example data setup creates workspaces called `example-workspace`, `example-workspace_1` etc. Would it be worth checking the airlock workspaces dir and also creating an empty workspace folder if there isn't a matching one already?

Reply:
Hmm, it should have created the airlock directory, will double check.